src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Pages/SignUp.razor.cs

```diff
@@ -26,7 +26,7 @@ public partial class SignUp
 
     private async Task DoSignUp()
     {
-        if (IsLoading)
+        if (IsLoading || SignUpModel.IsAcceptPrivacy is false)
```

Comment:
this is like a workaround. I don't like it.
shouldn't we try to validate the form instead?